src/current/_includes/molt/migration-prepare-database.md

```diff
@@ -23,4 +149,115 @@ For MySQL **5.7** sources, set the following values. Note that `binlog-row-image
 - `--binlog-row-image=full`
 - `--server-id={ID}`
 - `--log-bin=log-bin`
-</section>
\ No newline at end of file
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+##### Create source sentinel table
```

Comment:
Should we more clear that this only applies to Oracle? We don't need this sentinel table for other dialects because we don't buffer in mem.

Reply:
The `data-scope="oracle"` div ensures that this only appears when the Oracle tab is selected.